props/prop_base.h

```diff
@@ -419,8 +399,9 @@ class PropBase : CommandParser, Looper, protected SaberBase {
 #else
 #define WRAP_BLADE_SHORTERNER(N)
 #endif
+  }
 
-
+  void SetBladeStyle() {
```

Comment:
This name is a bit confusing, because "setting the blade style" would seem like it means setting the style string in the preset. Maybe ConfigureBladeStyles, or AllocateBladeStyles.

If we go with AllocateBladeStyles, then the function above should probably be called FreeBladeStyles()

Reply:
renamed AllocateBladeStyles and FreeBladeStyles